flake.nix

```diff
@@ -733,20 +743,28 @@
 
       devShells = let
         makeShell = pkgs: stdenv:
+          let
+            canRunInstalled = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
+          in
           with commonDeps { inherit pkgs; };
           stdenv.mkDerivation {
             name = "nix";
 
             outputs = [ "out" "dev" "doc" ];
 
             nativeBuildInputs = nativeBuildDeps
-                                ++ (lib.optionals stdenv.cc.isClang [ pkgs.bear pkgs.clang-tools ]);
+              ++ lib.optional stdenv.cc.isClang pkgs.buildPackages.bear
+              ++ lib.optional
+                (stdenv.cc.isClang && stdenv.hostPlatform == stdenv.buildPlatform)
+                pkgs.buildPackages.clang-tools
```

Comment:
What's the reason for this? `buildPackages` isn't the weird package set.

Reply:
Oh that might actually be fine. It was just doing a really slow build and I did not want to wait. (Because Nixpkgs regressed and now has a target specific LLVM build for no good reasons because the way it depends on binutils rrather than libbfd for LTO.)

Reply:
I'll do that in a follow-up PR,